src/api.rs

```diff
@@ -584,7 +586,9 @@ pub trait API: Send {
     ///
     /// `GET /api/v1/channels/get`
     ///
-    /// This call supports one or more GetterSelector.
+    /// This call supports one or more GetterSelector and gets the identifier
```

Comment:
"This call supports one or more GetterSelector. If the request itself requires authentication, the user is extracted from the HTTP request headers."

(same below)